nmdc_server/fakes.py

```diff
@@ -400,6 +401,10 @@ class Meta:
     }
     locked_by = None
     lock_updated = None
+    is_test_submission = False
```

Comment:
I noticed the other newly-added variables have a type hint (maybe only because those types aren't built-in Python types and aren't as implicit from their initial values).

```suggestion
    is_test_submission: bool = False
```